internal/fileutil/dir.go

```diff
@@ -26,8 +27,14 @@ func CopyAll(src, dst string) error {
 }
 
 func ClearDir(dir string) error {
+	f, err := os.Stat(dir)
+	if err != nil {
+		return errors.WithStack(err)
+	}
+	mode := f.Mode()
+
 	if err := os.RemoveAll(dir); err != nil {
 		return errors.WithStack(err)
 	}
-	return errors.WithStack(os.MkdirAll(dir, 0755))
+	return errors.WithStack(os.MkdirAll(dir, mode))
```

Comment:
what do you think of changing this function to not error if the directory doesn't exist?

In that case (i.e. not exist), we can `os.MkdirAll(dir, 0755)`

Reply:
I didn't think about it.
I should check if the `dir` exists first. If it doesn't, set mode to 0755. This check should be under L34.
I'll create a PR later to fix it.
Thx @savil

Reply:
See #1103